Uanion/Uanion.Application/MappingProfiles/ProfileMappingProfile.cs

```diff
@@ -8,6 +11,9 @@ public class ProfileMappingProfile : AutoMapper.Profile
     public ProfileMappingProfile()
     {
         CreateMap<CreateProfileCommand, Domain.Entities.Profile>();
+        CreateMap<UpdateProfileCommand, Profile>();
```

Comment:
Maybe we need a full class name specification for the "Profile" class?

Reply:
We can use a short class name specification for the "Profile" class in all cases.

Reply:
Done